packages/dnb-eufemia/src/components/tag/TagGroup.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import classnames from 'classnames'
+
+// Components
+import { createSpacingClasses } from '../space/SpacingHelper'
+
+// Shared
+import Context from '../../shared/Context'
+import { ISpacingProps } from '../../shared/interfaces'
+import { extendPropsWithContext } from '../../shared/component-helper'
+import Tag, { TagProps } from './Tag'
+
+export interface TagGroupProps {
+  /**
+   * Aria label to describe the tag group
+   * Default: null
+   */
+  label: string
+
+  /**
+   * Pass in a list of your pages as objects of tags.
```

Comment:
```suggestion
   * Pass in a list of your tags as objects of tags.
```
Not exactly sure, but pages sounded strange 😂

Reply:
Good catch, you are right😅
Updated this. Changed to `* Pass in a list of Tag objects.`